README.md

```diff
@@ -230,176 +247,156 @@ regexes instead. That is, if you are using a lexer, you should *not* use the
 
 ### EBNF
 
-nearley compiles some higher-level constructs into BNF for you. In particular,
-the `*`, `?`, and `+` operators from Regexes (or EBNF) are available as shown:
+nearley supports the `*`, `?`, and `+` operators from [EBNF](https://en.wikipedia.org/wiki/Extended_Backus–Naur_form) (or RegExps) as shown:
 
-    batman -> "na":* "batman" # nananana...nanabatman
+```ini
+batman -> "na":* "batman" # nananana...nanabatman
+```
 
 You can also use capture groups with parentheses. Its contents can be anything
 that a rule can have:
 
-    banana -> "ba" ("na" {% id %} | "NA" {% id %}):+
+```js
+banana -> "ba" ("na" {% id %} | "NA" {% id %}):+
+```
 
 ### Macros
 
-You can create "polymorphic" rules through macros:
+Macros allow to create "polymorphic" rules:
```

Comment:
Also, my mistake earlier, but "polymorphic" should either be replaced with a more appropriate word, or not be in scare quotes.